docs/examples/One_to_One_Associator.py

```diff
@@ -62,7 +67,7 @@
 plotter.fig
 
 # %%
-# This scenario has been created so it’s not immediately clear which states should be associated
+# This scenario has been created, so it’s not immediately clear which states should be associated
```

Comment:
```suggestion
# This scenario has been designed such the optimal association between ``states_from_a`` and ``states_from_b`` is unclear to the human eye.
```
The added comma here makes the phrasing confusing. The suggestion may need to go over a couple of lines.